### ISSUE TITLE
Create sigma-agnostic test metric

### ISSUE THREAD

**Issue by tlpss** [tlpss/box-manipulation#11]
Loss function is dependent on sigma, so create a score that does not depend on these details and captures the actual relevant metric. 

e.g. like the AP from COCO

**Comment by tlpss** [comment]
cf discussion Rembert & Peter

**Comment by tlpss** [comment]
- [ ] implement meanAP
- [ ] integrate meanAP in torchmetrics
- [ ] create validation loop that uses the meanAP on the validation set 
- [ ] do so for both classes
